config-linux.md

```diff
@@ -261,11 +261,14 @@ For more information, see the kernel cgroups documentation about [memory][cgroup
 
 Values for memory specify the limit in bytes, or `-1` for unlimited memory.
 
-* **`limit`** *(int64, OPTIONAL)* - sets limit of memory usage
-* **`reservation`** *(int64, OPTIONAL)* - sets soft limit of memory usage
+* **`limit`** *(int64, OPTIONAL)* - sets limit of memory usage.  It maps to `memory.limit_in_bytes` on cgroup v1 and to `memory.max` on cgroup v2
+* **`protection`** *(int64, OPTIONAL)* - sets hard memory protection.  It maps to `memory.min` on cgroup v2.  It can be used only on cgroup v2.
+* **`reservation`** *(int64, OPTIONAL)* - sets soft limit of memory usage.  It maps to `memory.soft_limit_in_bytes` on cgroup v1 and to `memory.low` on cgroup v2
+* **`softLimit`** *(int64, OPTIONAL)* - sets soft memory limit.  It maps to `memory.high` on cgroup v2.  It can be used only on cgroup v2.
```

Comment:
It might be beneficial to quote kernel docs, saying _this is the main mechanism to control memory usage. If usage goes over the high boundary, the processes of the container are throttled and put under heavy reclaim pressure_ or something like this.

Otherwise people will still use memory.limit which is not the best mechanism for v2.